projects/demo/src/modules/app/classes/ts-file-module.parser.ts

```diff
@@ -0,0 +1,25 @@
+import {TsFileParser} from './ts-file.parser';
+
+const DECLARATIONS_REG_EXP = /(?<=declarations:\s\[)(.*)(?=\])/gi;
+
+export class TsFileModuleParser extends TsFileParser {
+    addDeclaration(entity: string): void {
+        this.rawFileContent = this.rawFileContent.replace(
+            'declarations: [',
+            `declarations: [${entity}, `,
+        );
+    }
+
+    addModuleImport(entity: string): void {
+        this.rawFileContent = this.rawFileContent.replace(
+            'imports: [',
+            `imports: [\n${entity}, `,
+        );
+    }
+
+    hasDeclarationEntity(entityName: string): boolean {
```

Comment:
what difference between `entity: string` and `entityName: string`?

Reply:
could you, please, review all code at once ?)))))

and then write that you have finished review

Reply:
no difference, my mistake

Reply:
change it to just `entity `